<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearancePage</class>
 <widget class="QWidget" name="AppearancePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Style:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="paletteComboBox">
        <item>
         <property name="text">
          <string>Active palette</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Inactive palette</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disabled palette</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>82</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QMdiArea" name="mdiArea">
        <property name="tabsMovable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="styleComboBox"/>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="paletteGroupBox">
     <property name="title">
      <string>Palette</string>
     </property>
     <layout class="QVBoxLayout" name="_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="defaultPaletteButton">
          <property name="text">
           <string>Default</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="customPaletteButton">
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>230</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="colorScheeLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Color scheme:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="colorSchemeComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addSchemeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeSchemeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Change...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeSchemeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>customPaletteButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorScheeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPaletteButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorSchemeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPaletteButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>addSchemeButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPaletteButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>changeSchemeButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPaletteButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>removeSchemeButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
